### PR TITLE
pcs: bump to 0.9.159

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

File: components/cluster/pcs/Makefile [hunk 1]
```diff
@@ -15,14 +15,14 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= pcs
-COMPONENT_VERSION= 0.9.154
-COMPONENT_REVISION= 3
+COMPONENT_VERSION= 0.9.159
+COMPONENT_REVISION= 4
```
Comment (xen0l) [inline components/cluster/pcs/Makefile]: If COMPONENT_VERSION changes, you can drop COMPONENT_REVISION completely.

File: components/cluster/pcs/Makefile [hunk 2]
```diff
@@ -50,8 +52,8 @@ PKG_MACROS += RUBY_LIB_VERSION=$(RUBY_LIB_VERSION)
 COMPONENT_PRE_BUILD_ACTION = ($(RM) -r $(PROTO_DIR))
 
 COMPONENT_POST_BUILD_ACTION=( \
-    mkdir -p  $(PROTO_DIR)/usr/share/pcs/examples; \
-    mv $(PROTO_DIR)/etc/logrotate.d    $(PROTO_DIR)/usr/share/pcs/examples )
+    $(MKDIR)  $(PROTO_DIR)/usr/share/pcs/examples; \
+    $(MV) $(PROTO_DIR)/etc/logrotate.d    $(PROTO_DIR)/usr/share/pcs/examples )
```
Comment (xen0l) [inline components/cluster/pcs/Makefile:55]: Do we need to ship logrotate.d samples if the preferred tool is logadm?

File: components/cluster/pcs/files/pcsd.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0'?>
 <!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
 <service_bundle type='manifest' name='PCSD_WebGUI'>
-  <service name='application/pcs' type='service' version='1'>
+  <service name='application/pcsd' type='service' version='1'>
```
Comment (pyhalov) [inline components/cluster/pcs/files/pcsd.xml]: Why do we rename existing service? This can break users.

Comment (grueni) [reply components/cluster/pcs/files/pcsd.xml]: I realized that in linux systems the service has the name "pcsd".
And thought it would be better now to make this correction.
But I don't insist on the change.
Rollback to 'pcs'?

Comment (pyhalov) [reply components/cluster/pcs/files/pcsd.xml]: I'd say, yes.

File: components/cluster/pcs/Makefile (hunk 1 above)
Comment (pyhalov) [inline components/cluster/pcs/Makefile]: Remove COMPONENT_REVISION on COMPONENT_VERSION bump

File: components/cluster/pcs/pcs.p5m [hunk 4]
```diff
@@ -27,8 +28,8 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 depend type=require fmri=application/cluster/corosync
 depend type=require fmri=library/ruby/bundler
 
-file files/pcs.xml path=lib/svc/manifest/application/pcs.xml restart_fmri=svc:/system/manifest-import:default
-file files/svc-pcs path=lib/svc/method/svc-pcs
+file files/pcsd.xml path=lib/svc/manifest/application/pcs.xml restart_fmri=svc:/system/manifest-import:default
+file files/svc-pcsd path=lib/svc/method/svc-pcs
```
Comment (pyhalov) [inline components/cluster/pcs/pcs.p5m]: Where do you deliver README?

File: components/cluster/pcs/pcs.p5m [hunk 5]
```diff
@@ -2068,3 +2292,4 @@ file path=usr/sbin/pcs
 file path=usr/share/bash-completion/completions/pcs
 file path=usr/share/man/man1m/pcs.1m
 file path=usr/share/pcs/examples/logrotate.d/pcsd
+file files/README path=usr/share/pcs/examples/README
```
Comment (pyhalov) [inline components/cluster/pcs/pcs.p5m]: Please, move it to other custom files (to the top of the manifest), so that it wasn't accidentally removed on component update.

File: components/cluster/pcs/files/pcs.xml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0'?>
 <!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
 <service_bundle type='manifest' name='PCSD_WebGUI'>
-  <service name='application/pcs' type='service' version='1'>
+  <service name='application/pcsd' type='service' version='1'>
```
Comment (pyhalov) [inline components/cluster/pcs/files/pcs.xml]: revert this change, please

File: components/cluster/pcs/patches/010_man8.patch [hunk 7]
```diff
@@ -0,0 +1,94 @@
+--- pcs-0.9.159/pcs/usage.py.orig	2017-09-22 18:54:45.637013891 +0000
```
Comment (pyhalov) [inline components/cluster/pcs/patches/010_man8.patch]: please, use consistent names for patches NUMBER-NAME.patch (not NUMBER_NAME.patch) here and in other places

File: components/cluster/pcs/Makefile [hunk 8]
```diff
@@ -72,3 +74,4 @@ REQUIRED_PACKAGES += library/ruby/bundler
 REQUIRED_PACKAGES += runtime/python-27
 REQUIRED_PACKAGES += runtime/ruby-23
 REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/font/truetype/liberation
```
Comment (pyhalov) [inline components/cluster/pcs/Makefile]: Do I understand correctly, that this font is required during build time?

File: components/cluster/pcs/pcs.p5m (hunk 4 above)
Comment (pyhalov) [reply components/cluster/pcs/pcs.p5m]: Rename files back to pcs.xml/svc-pcs to avoid confusion.

File: components/cluster/pcs/Makefile (hunk 8 above)
Comment (grueni) [reply components/cluster/pcs/Makefile]: It is used in the webgui.

Comment (pyhalov) [reply components/cluster/pcs/Makefile]: Then likely it should be in package dependencies (depend type=require fmri=...), and not here.

File: components/cluster/pcs/pcs.p5m [hunk 9]
```diff
@@ -50,13 +52,23 @@ file path=usr/lib/pcsd/pcsd.conf
...
+file path=usr/lib/pcsd/pcsd_exchange_format.rb
+file path=usr/lib/pcsd/pcsd_file.rb
+file path=usr/lib/pcsd/pcsd_remove_file.rb
 file path=usr/lib/pcsd/permissions.rb
-file path=usr/lib/pcsd/public/css/LiberationSans-Bold.ttf
-file path=usr/lib/pcsd/public/css/LiberationSans-BoldItalic.ttf
-file path=usr/lib/pcsd/public/css/LiberationSans-Italic.ttf
-file path=usr/lib/pcsd/public/css/LiberationSans-Regular.ttf
+link path=usr/lib/pcsd/public/css/LiberationSans-Bold.ttf \
+    target=/usr/share/fonts/TrueType/liberation/LiberationSans-Bold.ttf
```
Comment (pyhalov) [inline components/cluster/pcs/pcs.p5m]: Targets are wrong!!!!

Comment (grueni) [reply components/cluster/pcs/pcs.p5m]: translates wrong to: use relative paths in links?
Or what do you mean?
The fonts must be linked into the folder with the css files.

**Review by pyhalov — APPROVED** [review]

**Comment by pyhalov** [comment]
I'm fine with this, but I'd like @xen0l also to take a look at this PR.

**Review by xen0l — APPROVED** [review]

File: components/cluster/pcs/Makefile (hunk 2 above)
Comment (pyhalov) [reply components/cluster/pcs/Makefile:55]: I think it doesn't hurt.
